src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <ul class="page__calendar calendar">
```

Comment:
```suggestion
    <ul class="page__calendar calendar calendar--28 calendar--mon">
```
Классы для установки первого дня и длины месяца должны работать на контейнере, а не на `calendar__day` элементах

Reply:
Исправил